### ISSUE TITLE
Missing switch seat_heating_activated

### ISSUE THREAD

**Issue by gerliczky** [skodaconnect/homeassistant-myskoda#280]
I have seat heating in my car. As far as I understood the seat_heating_activated switch should be added automatically, if the car has heated seats, howewer I se no switches for that. In My Skoda I see option to activate the front left and front right seats, so I guess these options should be available in the integration as well.

I have a Skoda Enyaq 80X, produced in 2023
SW version is 3.5

**Comment by WebSpider** [comment]
We dont have a seat heater switch in this integration yet. I guess this is now a feature request 😉

**Comment by prvakt** [comment]
just to have it here as well (recreated from decompiled myskoda.apk),
`HTTP POST api/v2/air-conditioning/{vin}/settings/seats-heating`
```
{
    "frontLeft": True/False,
    "frontRight": True/False,
    "rearLeft": True/False,
    "rearRight": True/False
}
```
and most likely also single seat is allowed in the body

I can implement it once I'm finished with the AuxiliaryHeater task

**Comment by prvakt** [comment]
#355 will add `Seat heating with with AC` switch ;)

**Comment by WebSpider** [comment]
This has been released with v1.12.0
